Automattic/jetpack
# Protect: Show vulnerability descriptions in VulAccordionItem

This PR adds vulnerability descriptions to Protect results.

#### Changes proposed in this Pull Request:

* Displays the vulnerability `description` when available, under a "What is the problem?" header.
* When a description is shown, displays the learn more link under the description. When no description is available, shows the learn more link at the very bottom of the accordion item.
* Updates the Jetpack Debug Protect Helper to include a "lorem ipsum" description in mock vulnerabilities.

#### Other information:


#### Jetpack product discussion



1201069996155224-as-1202476651392834/f

pbuNQi-37y-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Add a vulnerable plugin to your Protect report
* Visit the Protect admin page and validate the threat description is shown, and matches the layout in the screenshots below.

#### Screenshots

With description: 

<img width="636" alt="Screen Shot 2022-07-25 at 3 08 33 PM" src="https://user-images.githubusercontent.com/10933065/180875534-26b0e1c2-9cf0-4f53-a79b-fadf1c5182fc.png">

Without description:

<img width="635" alt="Screen Shot 2022-07-25 at 3 09 36 PM" src="https://user-images.githubusercontent.com/10933065/180875567-57f87b37-7a06-4650-b90a-393a68075f98.png">
